collect_weather_data_from_ghcn_bigquery.py

```diff
@@ -0,0 +1,89 @@
+"""
+Collect Weather Data from GHCN 
+Author: Francisco Vannini 
+Date: 2020-01-29
+
+Requires: 
+Google Project - For more information see : https://cloud.google.com/bigquery/docs/authentication/
+
+Inputs:
+cbc_cleaned_usa_statid.csv - csv file of cleaned cbc data matched with closest station id 
+
+Outputs: 
+flatten_data_from_<DATE>.csv - csv file of Percipitation, Temp low, Temp high and station ID and yer 
+
+"""
+import pandas as pd # Read in unfinished dataset
+from google.cloud import bigquery
+from google.cloud.bigquery.job import LoadJobConfig, WriteDisposition, CreateDisposition
+import pandas as pd
+
+
+# Change this to the PATH of your cbc_cleaned_usa_statid station
+df_shell = pd.read_csv('C:\\Users\\Frankie-Figz\\Downloads\\cbc_cleaned_usa_statid.csv')
```

Comment:
why not

```
df_shell = pd.read_csv('cbc_cleaned_usa_statid.csv')
```

We would need to make sure that the file `cbc_cleaned_usa_statid.csv` exists but it would work across all OS

Reply:
Will also include this change